### DISCUSSION TITLE
More Materials Compatibility

### DISCUSSION THREAD

**Discussion by Redgi110** [Roadhog360/Et-Futurum-Requiem#554] [Mod Support Suggestion]
[More Materials](https://legacy.curseforge.com/minecraft/mc-mods/more-materials) adds new wooden decorative blocks - Wooden Frames. I think it would be nice if EFR would add their variants from backported wood types.

**Comment by Roadhog360** [comment]
Does it have an API?

**Reply by Redgi110** [reply]
No as I know.

**Reply by Roadhog360** [reply]
Then it's very unlikely this will be fulfilled.
